src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -688,9 +688,30 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                             ->prototype('scalar')->end()
                         ->end()
                         ->arrayNode('enabled_locales')
-                            ->prototype('scalar')
+                            ->prototype('scalar')->end()
                             ->defaultValue([])
                         ->end()
+                        ->arrayNode('pseudo_localization')
+                            ->canBeEnabled()
+                            ->children()
+                                ->booleanNode('accents')->defaultTrue()->end()
+                                ->floatNode('expansion_factor')
+                                    ->validate()
+                                        ->ifTrue(static function (float $v): bool {
+                                            return $v < 1.0;
+                                        })
+                                        ->thenInvalid('The "expansion_factor" option must be greater than or equal to 1.')
```

Comment:
Change this to `->min(1.0)` instead of using a custom validation (this way, tooling could understand this rule too).

Reply:
Thanks I did not know this method.